### PR TITLE
postgresql: move to OpenSSL 1.1

### PR DESCRIPTION
**Pull request by igas**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/postgresql.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Postgresql < Formula
   url "https://ftp.postgresql.org/pub/source/v11.5/postgresql-11.5.tar.bz2"
   sha256 "7fdf23060bfc715144cbf2696cf05b0fa284ad3eb21f0c378591c6bca99ad180"
   head "https://github.com/postgres/postgres.git"
+  revision 1
```
Comment (fxcoudert) [inline Formula/postgresql.rb]: `revision` above `head`

**Comment by fxcoudert** [comment]
`citus` and `libpqxx` need revision bumped

**Comment by fxcoudert** [comment]
Also: there is at least one other formula that depends on both this and openssl: psqlodbc. So they should be switched together, to avoid psqlodbc having mixed dependencies.

**Comment by igas** [comment]
Thanks @scpeters for handling CI 👍

**Pull request closed without merging**